spec/features/staff/beis_users_can_create_a_programme_level_activity_spec.rb

```diff
@@ -13,6 +13,47 @@
     expect(page).to have_content t("action.programme.create.success")
   end
 
+  context "via a delivery partner's organisation page" do
+    let!(:delivery_partner) { create(:delivery_partner_organisation) }
+
+    before do
+      create(:fund_activity, :gcrf)
+      create(:fund_activity, :newton)
+    end
+
+    Fund.all.each do |source_fund|
+      context "with #{source_fund.name} as the funding source" do
+        scenario "reaches the 'roda_identifier' form step, with a newly created programme-level activity" do
+          visit organisations_path
+
+          within ".govuk-table__row##{delivery_partner.id}" do
+            click_on t("default.link.show")
+          end
+
+          click_on t("form.button.activity.new_with_source_fund", name: source_fund.name)
+
+          expect(page).to have_content t("form.label.activity.roda_identifier_fragment")
+
+          programme = Activity.programme.first
```

Comment:
Should this be `.last`? Just in case we have any others created for some reason

Reply:
Well, it'd have to be `Activity.order(:created_at).last`, because of the UUIDs..? The `CreateProgramme` service already tests that the defaults are set more thoroughly, so I think this is probably ok.